common/app/views/fragments/containers/facia_cards/container.scala.html

```diff
@@ -79,6 +81,29 @@
                         </div>
                     }
                 }
+
+                @defining((
+                    containerDefinition.index == 0,
+                    frontId.getOrElse(pageId)
+                    )) { case (isFirstContainer, id) =>
+
+                    @if(mvt.ABNewHeaderVariant.isParticipating && isFirstContainer) {
+
+                        <ul class="tertiary-navigation mobile-only">
+
+                            @NewNavigation.TertiaryNav.getTertiaryNavLinks(id).map { footballNavItem =>
+
+                                <li class="tertiary-navigation__item">
+                                    <a class="tertiary-navigation__link" href="@LinkTo(footballNavItem.url)" data-link-name="@footballNavItem.name">
+                                        @Html(footballNavItem.name)
+                                        <i class="tertiary-navigation__icon rounded-icon"></i>
```

Comment:
Could the icon be added as an `:after` on the `tertiary-navigation__link` instead of having its own empty element ?